contrib/dpu_daemon/scripts/build_arm.sh

```diff
@@ -2,63 +2,60 @@
 set -e
 
 arch=arm
-download=0
+download=${download:-0}
 build_ucx=1
 build_ucc=1
 build_mpi=1
 build_omb=0
 
-WORK_DIR='/global/scratch/users/dpu/deploy'
-BUILD_DIR="$WORK_DIR/build-$arch"
-
-UCX_SRC="$WORK_DIR/ucx"
-UCX_DIR="$BUILD_DIR/ucx"
-UCX_URL='https://github.com/openucx/ucx.git'
-UCX_BRANCH='master'
-
-UCC_SRC="$WORK_DIR/ucc"
-UCC_DIR="$BUILD_DIR/ucc"
-UCC_URL='https://github.com/Mellanox/ucc.git'
-UCC_BRANCH='v0.4.x'
-
-MPI_SRC="$WORK_DIR/ompi"
-MPI_DIR="$BUILD_DIR/ompi"
-MPI_URL='https://github.com/open-mpi/ompi.git'
-MPI_BRANCH='v5.0.x'
-
-OMB_SRC="$WORK_DIR/omb"
-OMB_DIR="$BUILD_DIR/omb"
-OMB_URL='https://github.com/paklui/osu-micro-benchmarks.git'
-OMB_BRANCH='master'
+DIR=$(dirname `realpath $0`)
+pushd ${DIR}
+. ./config.sh
 
 echo "#### WORK DIR  : $WORK_DIR  ####"
 echo "#### BUILD DIR : $BUILD_DIR ####"
+mkdir -p ${WORK_DIR}
```

Comment:
Please fix formatting and shellcheck issues

Reply:
Fixed